tuf/repository_lib.py

```diff
@@ -1980,14 +2019,15 @@ def write_metadata_file(metadata, filename, version_number,
  
   if consistent_snapshot:
     dirname, basename = os.path.split(written_filename)
-    
     basename = basename.split(METADATA_EXTENSION, 1)[0]
     version_and_filename = str(version_number) + '.' + basename + METADATA_EXTENSION 
     written_consistent_filename = os.path.join(dirname, version_and_filename)
 
-    logger.info('Linking ' + repr(written_consistent_filename))
-    os.link(written_filename, written_consistent_filename)
+    #logger.info('Linking ' + repr(written_consistent_filename))
+    #os.link(written_filename, written_consistent_filename)
```

Comment:
These two lines are commented out.  Why is this a copy, rather than a hard link?